docs/framework-introscope_agent.md

```diff
@@ -24,9 +24,8 @@ The credential payload of the service may contain the following entries:
 | Name | Description
 | ---- | -----------
 | `agent-name` | (Optional) The name that should be given to this instance of the Introscope agent
-| `host-name` | The host name of the Introscope Enterprise Manager server
-| `ssl` | (Optional) Whether or not to use an SSL connection to the Introscope Enterprise Manager server
-| `port` | (Optional) The port of the Introscope Enterprise Manager server
+| `url` | The url of the Introscope Enterprise Manager server
```

Comment:
No, we don't have any existing service brokers, so this will not be a problem for us.